exporter.py

```diff
@@ -8,19 +7,43 @@
 import json
 import re
 import sys
+import os
 
 from socketserver import ThreadingMixIn
 from http.server import HTTPServer
 from pprint import pformat
 import requests
 from prometheus_client import Gauge, Counter, Enum, MetricsHandler, core, Summary, start_http_server
-
+from urllib.parse import urlparse
+
+## Allow homematic_exporter to use the system-wide configured http(s) proxy
+class ProxiedTransport(xmlrpc.client.Transport):
+    def __init__(self):
+        proxy_http = os.environ.get('http_proxy')
+        proxy_https = os.environ.get('https_proxy')
+        proxy = proxy_https if proxy_https else proxy_http
```

Comment:
The fallback is mostly intended, but not helpful when only one scheme needs a proxy and the other doesn't. And the main decision should be taken on the originally requested scheme.